docs/docs/testing-your-assistant.mdx

```diff
@@ -117,113 +117,138 @@ Here are some examples:
+[f1-scores](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html)
+across all runs is plotted.
 
-The f1-score graph, along with all train/test sets, the trained models, classification and error reports, 
+The f1-score graph, along with all train/test sets, the trained models, classification and error reports,
 will be saved into a folder called `nlu_comparison_results`.
 
-### Intent Classification
+Inspecting the f1-score graph can help in understanding if you have enough data for your NLU model.
 
-The `rasa test` script will produce a report, confusion matrix, and confidence histogram for your model.
+You can also specify different number of runs and different exclusion percentages with:
+
+```bash {3}
+rasa test nlu --nlu data/nlu.yml
+  --config config_1.yml config_2.yml
+  --runs 4 --percentages 0 25 50 70 90
+```
 
-The report logs precision, recall and f1 measure for each intent and entity,
+### Interpreting the Output
+#### Intent Classifiers
+
+The `rasa test` script will produce a report, confusion matrix, and confidence histogram for your intent classification model.
+
+The report logs [precision](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_score.html),
+[recall](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.recall_score.html) and
+[f1-score](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html) for each intent,
 as well as providing an overall average. You can save these reports as JSON files using the `--report` argument.
 
-The confusion matrix shows you which intents are mistaken for others;
-any samples which have been incorrectly predicted are logged and saved to a file called `errors.json` for easier debugging.
+The confusion matrix shows which intents are mistaken for others.
+Any samples which have been incorrectly predicted are logged and saved to a file called `errors.json` for easier debugging.
 
-The histogram that the script produces allows you to visualise the confidence distribution for all predictions, with the volume of correct and incorrect predictions being displayed by blue and red bars respectively. Improving the quality of your training data will move the blue histogram bars to the right and the red histogram bars to the left of the plot.
+The histogram that the script produces allows you to visualize the confidence distribution for all predictions,
+with the volume of correct and incorrect predictions being displayed by blue and red bars respectively.
+Improving the quality of your training data will move the blue histogram bars to the right and the
+red histogram bars to the left of the plot.
 
-### Response Selection
+#### Response Selectors
 
-The evaluation script will produce a combined report for all response selector models in your pipeline.
+Similar to intent classifiers, the evaluation script outputs the same set of artifacts for response selectors -
```

Comment:
In addition to intent classifiers, `rasa test` also evaluates response selectors, producing a